projects/js-packages/shared-extension-utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@automattic/jetpack-shared-extension-utils",
-	"version": "0.9.0",
+	"version": "0.10.0-alpha",
```

Comment:
@jeherve What do we need to know about tagging a new release of this package? Should we take responsibility for tagging the release, or will someone else?

Reply:
A new version of the package will be automatically released next time a plugin that uses it needs to be released. In practice, that most likely means the next Jetpack weekly release.

In general, we don't need to worry about package releases; Ground Control takes care of them as they release plugins. We only need to make sure the dev version number is correct and matches the changes in the monorepo (like here; we're introducing new changes classified as minor to the package, so the version number must be bumped to a new minor version).